app/models/timer.rb

```diff
@@ -0,0 +1,23 @@
+class Timer < ActiveRecord::Base
+
+  belongs_to :player
+  belongs_to :game
+
+  def start!
+    @time_left = self.time_left
```

Comment:
This just makes the variable equal to itself & can likely be removed. `@` is basically a shortcut for `self`